packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -56,20 +56,36 @@ export default class TelecomPackMigrationOffersCtrl {
 
   updateOfferDisplayedPrice(value, offer, optionName) {
     let totalOfferPrice = offer.price.value;
+    let optionComfort = 0;
 
     angular.forEach(offer.options, (option) => {
-      if (option.name === 'gtr_ovh' && option.selected) {
-        totalOfferPrice += option.optionalPrice.value;
+      if (
+        option.name.match(/^gtr_/) &&
+        option.selected !== null &&
+        optionName.match(/^gtr_/)
+      ) {
+        let val = option.selected === true ? 1 : 0;
+        if (option.name !== optionName) {
+          option.selected = false; // eslint-disable-line no-param-reassign
+          val = 0;
+        } else if (optionName.match(/^gtr_\d{1,2}m_/)) {
+          optionComfort = 1;
+        }
+        totalOfferPrice += val * option.optionalPrice.value;
       } else if (option.name === optionName) {
         totalOfferPrice += value * option.optionalPrice.value;
+      } else if (`${option.name}_${option.optional}` === optionName) {
```

Comment:
this `else if` condition could be merged with the previous one

Reply:
right, done